DSharpPlus/Entities/AuditLogs/AuditLogParser.cs

```diff
@@ -96,6 +101,7 @@ AuditLog auditLog
         IOrderedEnumerable<AuditLogAction>? auditLogActions = auditLog.Entries.OrderByDescending(xa => xa.Id);
         foreach (AuditLogAction? auditLogAction in auditLogActions)
         {
+            if (cancellationToken.IsCancellationRequested) yield break;
```

Comment:
style, should be a full if statement with braces and a following newline after the block